src/content/docs/service-level-management/create-slm.mdx

```diff
@@ -18,9 +18,11 @@ import slmSummaryEditButton from 'images/slm_screenshot-crop_summary-edit-button
 
 You can create SLIs and SLOs manually through the [New Relic UI](https://one.newrelic.com). Alternatively, you can automate the process with our [NerdGraph API](/docs/apis/nerdgraph/examples/nerdgraph-slm/) and the [Terraform Service Level resource](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/service_level).
 
-## Requirements [#requirements]
+## Requirements and limitations [#requirements]
 
-You must have the [capability for modifying and deleting events-to-metrics](/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities#insights).
+To create and manage workloads, a New Relic user requires the [capability for modifying and deleting events-to-metrics](/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities#insights).
```

Comment:
@zuluecho9 this link for events-to-metrics takes me to a glossary entry for Insights. How about just sending users directly here: https://docswebsitedevelop-workloadsserviceclarification.gtsb.io/docs/data-apis/convert-to-metrics/analyze-monitor-data-trends-metrics

Reply:
that's not a glossary entry for 'insights'. unfortunately that's reflecting the unfortunate reality that 'Insights' is still used in the user mgmt 'capabilities' UI.